rl/function_approx.py

```diff
@@ -36,13 +47,22 @@ def within(self, other: FunctionApprox[X], tolerance: float) -> bool:
         '''
         pass
 
-    @staticmethod
-    def converged(iterator: Iterator[FunctionApprox[X]],
-                  tolerance: float = 0.0001) -> FunctionApprox[X]:
-        def done(a, b):
-            return a.within(b, tolerance)
+    def rmse(
+        self,
+        xy_seq: Sequence[Tuple[X, float]]
+    ) -> float:
+        x_seq, y_seq = zip(*xy_seq)
+        errors: np.ndarray = self.evaluate(x_seq) - np.array(y_seq)
+        return np.sqrt(np.mean(errors * errors))
 
-        return iterate.converged(iterator, done=done)
+    def iterate_updates(
```

Comment:
This is another function that could really use a docstring.

Reply:
Added docstring, as you suggested